en/complete_vehicles/atl_mantis_edu.md

```diff
@@ -0,0 +1,251 @@
+
+The command to upload a file is:
+```sh
+curl -F "image=@$HOME/path/to/file.ext" -H "Expect:" http://192.168.42.1/cgi-bin/upload
+```
+
+For bigger (multi-part) upload, use:
+```sh
+curl -F "image=@$HOME/path/to/big-file.ext" -H "Expect:" -H "File-Size: $(stat -c%s $HOME/path/to/big-file.ext)" http://192.168.42.1/cgi-bin/upload
+```
+
+### RTSP Connection
+
+Video is streamed at 1280x720 at 30 Hz over RTSP.
+QGC should connect automatically, as Mantis broadcasts the stream parameters using the [VIDEO_STREAM_INFORMATION](https://mavlink.io/en/messages/common.html#VIDEO_STREAM_INFORMATION) MAVLink message.
+
+If you want to connect manually, you can do so on `rtsp://192.168.42.1:554/live`.
+
+
+## Troubleshooting/FAQ
+
+### MAVLink and the camera don't connect even though WiFi is connected
+
+Try resetting the camera by pressing a paperclip into the small hole above the SD card slot for 5 seconds.
+The camera should reboot and reset some of its internal settings.
+
+### Why does it take over a minute for the WiFi / connection to appear?
+
+The Mantis uses DFS channels on 5 GHz.
+In order to be allowed to use these channels, the wifi module needs to listen on these frequency bands for some time to make sure they are not occupied by radars, and other official users.
+Once it is verified that the bands are unoccopied, the WiFi will start.
+
+TODO: check if this is required in the US
+
+### The WiFi does not come up
+
+If the WiFi does not come up after a minute or so, the most common cause is that USB is connected (you can either connect WiFi or USB, but not both at the same time).
+
+### The beeps/sound are not working during flight
+
+The beeps are generated by the motors.
+Therefore, if the motors are spinning, it's not possible to generate beeps.
+
+### How can I switch between the internal storage and the SD card?
+
+The primary storage can be selected in QGC.
+Connect to drone via WiFi, start QGC, and go to the camera settings (gear icon next to camera buttons on the right of the screen).
+There, you should find the option to select the storage.
+
+### Can I upload the PX4 firmware by connecting over USB?
+
+No, this is not possible, unless you have a special cable.
+
+You can however, copy the `build/atl_mantis-edu_default/atl_mantis-edu_default.px4` file on SD card and rename it to `autopilot.px4`.
+If you then power up the Mantis, it will flash the file automatically.
+
+## What is open-source?
+
+- The PX4 firmware is open-source.
+- The camera is proprietary but implementing the [MAVLink camera protocol](https://mavlink.io/en/services/camera.html) and [MAVLink camera definition](https://mavlink.io/en/services/camera_def.html).
+- The gimbal is proprietary but implementing the [MAVLink gimbal v2 protocol](https://mavlink.io/en/services/gimbal_v2.html).
+- The flow module is proprietary communicating via MAVLink.
+
+## How to get help/technical support?
+
+For hardware issues, please contact TODO: add vendor.
```

Comment:
TODO: We need to provide information on the Vendor and also where to buy the Mantis Edu

Reply:
We'll do it later.